src/toolkits/drawing_classifier/drawing_classifier.hpp

```diff
@@ -6,53 +6,56 @@
 #ifndef TURI_DRAWING_CLASSIFIER_H_
 #define TURI_DRAWING_CLASSIFIER_H_
 
-#include <core/logging/table_printer/table_printer.hpp>
-#include <model_server/lib/extensions/ml_model.hpp>
-#include <model_server/lib/variant.hpp>
 #include <core/data/sframe/gl_sframe.hpp>
-#include <toolkits/coreml_export/mlmodel_wrapper.hpp>
+#include <core/logging/table_printer/table_printer.hpp>
 #include <ml/neural_net/compute_context.hpp>
 #include <ml/neural_net/model_backend.hpp>
 #include <ml/neural_net/model_spec.hpp>
+#include <model_server/lib/variant.hpp>
+#include <model_server/lib/extensions/ml_model.hpp>
+#include <toolkits/coreml_export/mlmodel_wrapper.hpp>
 
 namespace turi {
 namespace drawing_classifier {
 
-class EXPORT drawing_classifier: public ml_model_base {
-
+class EXPORT drawing_classifier : public ml_model_base {
  public:
-  
   // ml_model_base interface
 
   /* Commented out for the purpose of a skeleton. */
-  // void init_options(const std::map<std::string, flexible_type>& opts) override;
-  // size_t get_version() const override;
-  // void save_impl(oarchive& oarc) const override;
-  // void load_version(iarchive& iarc, size_t version) override;
+  // void init_options(const std::map<std::string, flexible_type>& opts)
+  // override; size_t get_version() const override; void save_impl(oarchive&
+  // oarc) const override; void load_version(iarchive& iarc, size_t version)
+  // override;
```

Comment:
I don't think this improves the comment

Reply:
that must be my IDE's auto-indent. Will clean it up.